blueocean-web/src/main/resources/io/jenkins/blueocean/TryBlueOceanPageDecorator/header.jelly

```diff
@@ -0,0 +1,8 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+  <!--
+  The following Stapler adjunct adds the "Try Blue Ocean JavaScript bundle to
+  every page in "classic" Jenkins.
+  -->
+  <st:adjunct includes="io.jenkins.blueocean.try"/>
```

Comment:
:ant: This whole thing would be easier to just have a link instead of including js modules for it on every page...